assets/queries/azureResourceManager/sql_server_database_with_low_retention_days/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "c09cdac2-7670-458a-bf6c-efad6880973a",
+  "queryName": "SQL Server Database With Low Retention Days",
```

```suggestion
  "queryName": "SQL Server Database With Inadequate Retention Days",
```

```suggestion
  "queryName": "SQL Server Database With Unrecommended Retention Days",
```

which one is better?